src/main/java/org/testng/internal/PackageUtils.java

```diff
@@ -30,7 +30,7 @@
   private static String[] s_testClassPaths;
 
   /** The additional class loaders to find classes in. */
-  private static final List<ClassLoader> m_classLoaders = new Vector<ClassLoader>();
+  private static final List<ClassLoader> m_classLoaders = new Vector<>();
```

Comment:
Since TestNG is Java 5 compatible, I'd suggest to use java 5/6 sources level.
BTW All modern IDE presents that non-sugar code as simple diamond.